extensions/amp-analytics/amp-analytics.md

```diff
@@ -137,6 +137,7 @@ The configuration object for `<amp-analytics>` uses the following format:
     request-name: request-value,
     ...
   },
+  "requestOrigin": requestOrigin-value,
```

Comment:
'requestOrigin' is documented in the section below. I don't think we should include it here because it is an optional field.